backend/app/controllers/spree/admin/products_controller.rb

```diff
@@ -6,6 +6,7 @@ class ProductsController < ResourceController
       helper 'spree/products'
 
       before_action :load_data, except: [:index]
+      before_action :split_taxons, :split_option_types, only: [:create, :update]
```

Comment:
I'd prefer splitting these in two lines, so git history is easier to read when these change. But it's just a non-blocking preference 🙂